packages/docs/src/matchers/symmetric/index.ts

```diff
@@ -0,0 +1,63 @@
+import { Function } from "../../interfaces";
+
+import * as _toBe from "./toBe.json";
+import * as _toBeCloseTo from "./toBeCloseTo.json";
+import * as _toBeDefined from "./toBeDefined.json";
+import * as _toBeFalsy from "./toBeFalsy.json";
+import * as _toBeGreaterThan from "./toBeGreaterThan.json";
+import * as _toBeGreaterThanOrEqual from "./toBeGreaterThanOrEqual.json";
+import * as _toBeInfinity from "./toBeInfinity.json";
+import * as _toBeInstanceOf from "./toBeInstanceOf.json";
+import * as _toBeLessThan from "./toBeLessThan.json";
+import * as _toBeLessThanOrEqual from "./toBeLessThanOrEqual.json";
+import * as _toBeNaN from "./toBeNaN.json";
+import * as _toBeTruthy from "./toBeTruthy.json";
+import * as _toBeUndefined from "./toBeUndefined.json";
+import * as _toContain from "./toContain.json";
+import * as _toContainEqual from "./toContainEqual.json";
+import * as _toEqual from "./toEqual.json";
+import * as _toHaveBeenCalled from "./toHaveBeenCalled.json";
+import * as _toHaveBeenCalledTimes from "./toHaveBeenCalledTimes.json";
+import * as _toHaveBeenCalledWith from "./toHaveBeenCalledWith.json";
+import * as _toHaveBeenLastCalledWith from "./toHaveBeenLastCalledWith.json";
+import * as _toHaveBeenNthCalledWith from "./toHaveBeenNthCalledWith.json";
+import * as _toHaveReturned from "./toHaveReturned.json";
+import * as _toHaveReturnedTimes from "./toHaveReturnedTimes.json";
+import * as _toHaveReturnedWith from "./toHaveReturnedWith.json";
+import * as _toHaveLastReturnedWith from "./toHaveLastReturnedWith.json";
+import * as _toHaveNthReturnedWith from "./toHaveNthReturnedWith.json";
+import * as _toHaveLength from "./toHaveLength.json";
+import * as _toMatch from "./toMatch.json";
+import * as _toMatchObject from "./toMatchObject.json";
+import * as _toThrow from "./toThrow.json";
+
+export const toBe: Function = _toBe;
```

Comment:
Is there no way to retype the export inline with the above?

i.e. something equivalent to:
```ts
export { default as blah } from "./blah.json" as Function;
```

Reply:
Not that I know of.

1. require does not resolve the types at compile, so we lose our strict typechecking of the docs.
2. `export { default as blah from "./blah" }` is unable to be typed.

Reply:
Looked into it, can't find a way